proxy/src/main/java/com/velocitypowered/proxy/plugin/VelocityEventManager.java

```diff
@@ -51,11 +52,11 @@ public void register(@NonNull Object plugin, @NonNull Object listener) {
     }
 
     @Override
-    public <E> void register(@NonNull Object plugin, @NonNull Class<E> eventClass, @NonNull EventHandler<E> handler) {
+    public <E> void register(@NonNull Object plugin, @NonNull Class<E> eventClass, @NonNull PostOrder postOrder, @NonNull EventHandler<E> handler) {
         Preconditions.checkNotNull(plugin, "plugin");
         Preconditions.checkNotNull(eventClass, "eventClass");
         Preconditions.checkNotNull(handler, "listener");
-        bus.register(eventClass, new KyoriToVelocityHandler<>(handler));
```

Comment:
Plugins may not respect checker-qual, so make sure to add in an explicit null check for the `postOrder`.